### ISSUE TITLE
SKIP_GIT ignored in build.sh

### ISSUE THREAD

**Issue by mattolenik** [netbox-community/netbox-docker#190]
## Current Behavior

When running `build.sh`, NetBox gets re-cloned into `.netbox` regardless if the `SKIP_GIT` environment variable is defined. As per the usage docs, this should not be the case:
```
  echo "  SKIP_GIT    If defined, git is not invoked and \${NETBOX_PATH} will not be altered."
  echo "              This may be useful, if you are manually managing the NETBOX_PATH."
  echo "              Default: undefined"
```

The string `SKIP_GIT` isn't actually referenced anywhere, that one line is the only occurrence in the repo.

## Expected Behavior
`.netbox` should not be modified.

**Comment by mattolenik** [comment]
Solution proposed in #191

**Comment by cimnine** [comment]
Fixed in #191.
